go.sum

```diff
@@ -1,7 +1,10 @@
+bou.ke/monkey v1.0.1 h1:zEMLInw9xvNakzUUPjfS4Ds6jYPqCFx3m7bRmG5NH2U=
```

Comment:
Why are there changes here without changes to `go.mod`?

Reply:
I ran `go mod tidy`